consensus/consensus-types/src/sync_info.rs

```diff
@@ -72,6 +72,10 @@ impl SyncInfo {
             .map_or(0, |tc| tc.round())
     }
 
+    pub fn hcc_round(&self) -> Round {
```

Comment:
why not just call this highest_commit_round?

Reply:
probably lazy? I'll swap them with full name